Makefile.toml

```diff
@@ -87,6 +85,12 @@ BUILDSYS_VERSION_FULL="${BUILDSYS_VERSION_IMAGE}-${BUILDSYS_VERSION_BUILD}"
 BUILDSYS_NAME_VARIANT="${BUILDSYS_NAME}-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}"
 BUILDSYS_NAME_VERSION="${BUILDSYS_NAME}-${BUILDSYS_VERSION_FULL}"
 BUILDSYS_NAME_FULL="${BUILDSYS_NAME_VARIANT}-${BUILDSYS_VERSION_FULL}"
+# Relative path to repo-specific root role.  This gets mapped into the package
+# build environment so it can be included in the image, and Docker only accepts
+# relative paths for bind mounts.
+PUBLISH_REPO_ROOT_JSON_RELATIVE = "roles/${PUBLISH_REPO}.root.json"
```

Comment:
Good catch; I'm not sure we need this to be relative anymore, now that we're COPYing the file rather than mapping.  It also really bugged me, but I forgot to go back to it after switching back to COPY.

Reply:
After discussion, we're going to pass through PUBLISH_REPO instead, which we can use in the COPY line, and we'll form the (relative) path in the COPY call instead of passing the relative path through the whole way.